ledger-service/http-json/src/main/scala/com/digitalasset/http/HealthService.scala

```diff
@@ -4,37 +4,49 @@
 package com.daml.http
 
 import akka.http.scaladsl.model.{ContentTypes, HttpEntity, HttpResponse, StatusCodes}
+import io.grpc.health.v1.health.HealthCheckResponse
 import scalaz.Scalaz._
 
 import scala.concurrent.{ExecutionContext, Future}
-import scala.util.Try
+import scala.util.{Failure, Success}
 import scala.util.control.NonFatal
 
 final class HealthService(
-    getLedgerEnd: HealthService.GetLedgerEnd,
+    getLedgerEnd: HealthService.GetHealthCheckResponse,
```

Comment:
Probably rename this?

Reply:
good point c12384a11cd0ad4a0f447f896f3e9bb5d3425e54